### ISSUE TITLE
Modeling Weights/ Data Processing Script

### ISSUE THREAD

**Issue by isabelcachola** [maszhongming/MatchSum#2]
Super interesting work! Can you please release the model weights? Also a data preprocessing script so we can try your model on other datasets. Thanks!

**Comment by maszhongming** [comment]
Thank you for your interest in our work. We are willing to help you try our model on other datasets, we will release the model weights and processing script you need within one week.

**Comment by shreydesai** [comment]
Any update on this?

**Comment by maszhongming** [comment]
We have released the pre-trained models and processing files now.  Please feel free to contact me if you have any questions.

**Comment by shreydesai** [comment]
Thanks! By any chance, do you have preprocessed files for other datasets evaluated in the paper, like WikiHow, Pubmed, etc.?

**Comment by maszhongming** [comment]
All our datasets use the same processing files, and we will release all processed datasets soon.

**Comment by maszhongming** [comment]
Now, we have released all the datasets.

**Comment by shreydesai** [comment]
Thanks! One small question -- what does `label` represent in each file? Is this your oracle or the original BERTSum greedy extractive oracle?

**Comment by maszhongming** [comment]
`label` is the greedy extractive oracle we use to train our BertExt.
